website_medical_prescription_order_line/views/medical_prescription_order_line_template.xml

```diff
@@ -19,83 +18,60 @@
+
+                                <label class="label label-info">Remain</label>
+                                <span t-field="line.can_dispense_qty" /><br/>
+
                                 <t t-if="line.last_dispense_id.date_planned">
-                                   <span t-field="line.last_dispense_id.date_planned" t-field-options='{"widget": "date"}' />
+                                    <label class="label label-info">Last Disp</label><br/>
+                                    <span t-field="line.last_dispense_id.date_planned" t-field-options='{"widget": "date"}' />
                                 </t>
-                            </td>
-                            <td colspan="2">
+
                                 <t t-if="line.date_stop_treatment">
-                                   <span t-field="line.date_stop_treatment" t-field-options='{"widget": "date"}' />
+                                    <label class="label label-info">Expiration</label><br/>
+                                    <span t-field="line.date_stop_treatment" t-field-options='{"widget": "date"}' />
                                 </t>
-                            </td>
-                            <td colspan="1" class="text-right">
+
                                 <t t-if="line.can_dispense_qty">
-                                    
                                     <t t-set="product" t-value="line.medicament_id.product_id" />
-                                    
                                     <form action="/shop/cart/update" method="POST">
                                         <input type="hidden"
-                                               name="csrf_token"
-                                               t-att-value="request.csrf_token()"
-                                               />
+                                                name="csrf_token"
+                                                t-att-value="request.csrf_token()"
+                                                />
                                         <input type="hidden"
-                                               class="product_id"
-                                               name="product_id"
-                                               t-att-value="int(product.product_variant_ids[0]) if len(product.product_variant_ids) == 1 else '0'"
-                                               />
+                                                class="product_id"
+                                                name="product_id"
+                                                t-att-value="int(product.product_variant_ids[0]) if len(product.product_variant_ids) == 1 else '0'"
+                                                />
                                         <input type="hidden" name="set_qty" t-att-value="line.qty" />
                                         <t t-if="line.id in order_lines_filtered.ids or [line.id] == order_lines_filtered.ids">
```

Comment:
Please add a note to readme:
* Refactor around requirement for pre-existing Sale Order for Add to Cart button